metricbeat/module/vsphere/virtualmachine/virtualmachine.go

```diff
@@ -138,8 +138,13 @@ func (m *MetricSet) Fetch() ([]common.MapStr, error) {
 
 			freeMemory := (int64(vm.Summary.Config.MemorySizeMB) * 1024 * 1024) - (int64(vm.Summary.QuickStats.GuestMemoryUsage) * 1024 * 1024)
 
+			var host string
+			if vm.Summary.Runtime.Host != nil {
+				host = vm.Summary.Runtime.Host.Value
```

Comment:
Same comment as above. Should we log Error or Debug here and do we know the reason it can be nil?

Reply:
Same as above, `Host` withing `Runtime` is a pointer but the reason why it might be `nil` is unknown

Reply:
See above for the Debug messages.

Reply:
Extended information added as debug messages.